### PR TITLE
init prow jobs for acm cluster-proxy-addon

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: liuwei <liuweixa@redhat.com>

### CODE REVIEW

**Comment by skeeey** [comment]
refer to https://github.com/open-cluster-management/backlog/issues/14761

File: ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-main.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        --token-path=/etc/github/token \
+        --current-promotion-namespace=open-cluster-management \
+        --current-release=2.4 \
+        --future-release=2.4
+  container:
+    from: repo-brancher
+  postsubmit: true
+  secrets:
+  - mount_path: /etc/github
+    name: acm-cicd-github
```
Comment (ray-harris) [inline ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-main.yaml:80]: We have a new way to fast forward that's still being documented. Replace this fast forward with this:

```
- as: fast-forward
  postsubmit: true
  steps:
    env:
      DESTINATION_BRANCH: release-2.4
    workflow: ocm-ci-fastforward
```

**Review by ray-harris — CHANGES_REQUESTED** [review]
In both the main and release-2.4 branches, you need both the sonar-pre-submit and the sonar-post-submit jobs:

https://github.com/open-cluster-management/cicd-docs/blob/main/prow/onboarding.md#go-sonarcloud-pre-submit-scan

https://github.com/open-cluster-management/cicd-docs/blob/main/prow/onboarding.md#go-sonarcloud-post-submit-scan

File: ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-main.yaml (hunk 1 above)
Comment (skeeey) [reply ci-operator/config/open-cluster-management/cluster-proxy-addon/open-cluster-management-cluster-proxy-addon-main.yaml]: cool, updated

**Comment by skeeey** [comment]
@ray-harris I added the both sonar jobs, but the pre job is failed, did I miss some things? please help me to take a look, thanks

**Pull request merged**